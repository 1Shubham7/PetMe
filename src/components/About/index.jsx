import React from 'react';

function index() {
  return (
    <section className="bg-[#111827]">
      <section className="max-w-screen-xl mx-auto p-6 flex flex-col items-center">
        <h1 className="text-white font-bold text-3xl lg:text-5xl py-4">
          About
        </h1>
        <p className="text-[#696C72] text-center pb-4 text-sm lg:text-xl">
          Our mission is to provide a platform that connects animal lovers with pets in need
          of a forever home,
          while also providing emergency medical care for stray animals in distress.
          We strive to make a difference in the lives of these innocent creatures by
          offering options to Adopt,
          Donate and through our SOS feature Report stray animals in need of immediate assistance.
          Together, we can create a better world for all animals.
        </p>
      </section>
    </section>
  );
}

export default index;
