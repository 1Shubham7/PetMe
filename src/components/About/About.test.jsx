import { render } from '@testing-library/react';
import React from 'react';
import About from '.';

describe('About', () => {
  test('renders About', () => {
    const { getByRole } = render(<About />);
    const heading = getByRole('heading', { name: /about/i });
    expect(heading).toBeInTheDocument();
  });

  test('renders About with the right text', () => {
    const { getByText } = render(<About />);
    const text = getByText(
      /Our mission is to provide a platform that connects animal lovers with pets in need of a forever home, while also providing emergency medical care for stray animals in distress. We strive to make a difference in the lives of these innocent creatures by offering options to Adopt, Donate and through our SOS feature Report stray animals in need of immediate assistance. Together, we can create a better world for all animals./i,
    );
    expect(text).toBeInTheDocument();
  });
});
