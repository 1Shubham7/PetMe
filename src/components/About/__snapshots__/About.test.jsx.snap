// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`About > renders About with snapshot 1`] = `
<DocumentFragment>
  <section
    class="bg-[#111827]"
  >
    <section
      class="max-w-screen-xl mx-auto p-6 flex flex-col items-center"
    >
      <h1
        class="text-white font-bold text-5xl py-4"
      >
        About
      </h1>
      <p
        class="text-[#696C72] text-center pb-4"
      >
        Our mission is to provide a platform that connects animal lovers with pets in need of a forever home, while also providing emergency medical care for stray animals in distress. We strive to make a difference in the lives of these innocent creatures by offering options to Adopt, Donate and through our SOS feature Report stray animals in need of immediate assistance. Together, we can create a better world for all animals.
      </p>
    </section>
  </section>
</DocumentFragment>
`;
